doc/sphinx/source/features.rst

```diff
@@ -18,8 +18,7 @@ Frontend/Clang
   * pipes (WIP)
   * device-side enqueue
 
-* cl_khr_f16: half precision float literals
-
+* cl_khr_f16: half precision support (with the exception of  vload_half / vstore_half)
```

Comment:
What was the problem with vloads?

Reply:
The problem is not with vload halfs, the problem is with everything else (this is the list of unsupported features). Basically halfs on CPU are completely unsupported with the exception of vload_half and vstore_half.

Reply:
Ah, right.